crates/bevy_gltf/src/loader.rs

```diff
@@ -756,18 +751,15 @@ fn load_node(
                 Projection::Perspective(perspective_projection)
             }
         };
-
-        node.insert((
+        node.insert(Camera3dBundle {
             projection,
-            Camera {
+            transform,
```

Comment:
Does setting the camera's transform here make sense when you've already set the parent node to the same transform?

The results look correct, but IMO it feels wrong.

Reply:
It's not the parent node that has the same transform, it's the camera node itself. The `Camera3dBundle` is inserted after the `SpatialBundle`, and because both have a `Transform` component, the camera bundle overwrites the transform of the spatial bundle. So we use the same transform twice.

Reply:
It feels wrong, but there aren't currently a way to add a "partial" bundle. The other way to fix this would be to add manually the component of the bundle specific to the camera, but then it will crash again next time we add a new mandatory component for cameras.